website/models.py

```diff
@@ -312,6 +313,26 @@ def remove_user(self):
     def get_absolute_url(self):
         return "/issue/" + str(self.id)
 
+    def save_cve_score(self):
```

Comment:
I liked the old name better. The current name doesn't reflect what the method does (technically it doesn't save the score but rather gets it via API and puts into `self.score`).

Reply:
ohkay !!